guide/README.md

```diff
@@ -8,30 +8,36 @@ For now, TDS is intended for use by TELUS employees and approved partners.
 
 To view recent and past announcements, see our [Announcements](announcements.html) page.
 
-## TDS V2: Split Components to V3: styled-components migration
-
-TDS V3: styled-components is due to arrive on **July 1st, 2019**! Check out our [migration guide](v3-upgrade.md) for information on how to prepare your application for the upgrade from V2: Split Components.
-
 ## Important links
 
-### New to TDS?
+### Getting started guides
 
 If you're new to **utilizing** the Telus Design System, check out the [getting started guide](ref://getting-started/getting-started.html).
 
-### Contributing?
+### Contribution guides
 
 If you are a **developer** who wants to contribute to TDS, please see the [developer contributor guide](ref://contributing/developer-guide.html).
 
 If you are a **designer** who wants to contribute to TDS, please see the [designer contributor guide](ref://contributing/designer-guide.html).
 
-### References
+### Migration guides
 
-To view the **technical documentation** for the components, see the [component catalogue](ref://components/index.html).
+#### TDS V2: Split Components to V3: Styled Components migration
 
-To view **documentation from a previous version**, copy this URL into your browser's address bar, changing the version at the end to the desired version: <http://telus-design-system-docs.s3-website-us-east-1.amazonaws.com/v0.34.0/>.
+TDS V3: Styled Components is due to arrive on **July 1st, 2019**! Check out our [migration guide](v3-upgrade.md) for information on how to prepare your application for the upgrade from V2: Split Components.
 
-If you're on V0.y.z of `@telusdigital/tds`, please follow our migration guide for [upgrading to v1.y.z](https://github.com/telusdigital/tds-core/releases/tag/v1.0.0).
+#### TDS V1.0.0 to V2: Split Components migration
 
 If you're already on V1.y.z of `@telusdigital/tds`, please follow our migration guide for [upgrading to split components](https://github.com/telusdigital/tds-core/releases/tag/v2.0.0).
 
+#### TDS V0.y.z to V1.0.0 migration
+
+If you're on V0.y.z of `@telusdigital/tds`, please follow our migration guide for [upgrading to v1.y.z](https://github.com/telusdigital/tds-core/releases/tag/v1.0.0).
```

Comment:
If we're sticking with a capital V everywhere for versions, we should probably change "upgrading to v1.y.z" to "upgrading to V1.y.z"